### PR TITLE
yams: init at 0.7.3

### PR DESCRIPTION
**Pull request by ccellado**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/yams/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ lib, fetchgit, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "yams";
+  version = "0.7.3";
+
+  src = fetchgit {
```
Comment (sikmir) [inline pkgs/applications/audio/yams/default.nix:7]: Use `fetchFromGitHub`.

File: pkgs/applications/audio/yams/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/Berulacks/yams";
+    description = "Last.FM scrobbler for MPD";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ ccellado ];
+    platforms = platforms.all;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/yams/default.nix:28]:
```suggestion
```

set by buildPythonPackage

File: pkgs/applications/audio/yams/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    mpd2
+    requests
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/Berulacks/yams";
+    description = "Last.FM scrobbler for MPD";
+    license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/yams/default.nix:26]: gpl3Plus or gpl3Only?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/yams/default.nix (hunk 3 above)
Comment (ccellado) [reply pkgs/applications/audio/yams/default.nix:26]: Oh, its relatively new. gpl3Only then.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110887` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>yams</li>
</ul>
</details>

**Review by sikmir — APPROVED** [review]

**Pull request merged**